src/elements/content-uploader/ItemList.js

```diff
@@ -56,7 +65,6 @@ const ItemList = ({ isResumableUploadsEnabled = false, items, onClick, onRemoveC
                         cellRenderer={actionCell}
                         dataKey="status"
                         flexShrink={0}
-                        width={25}
```

Comment:
removed to allow the upgrade button to show full width. See added screenshots for behavior changes

Reply:
The `react-virtualized` types indicate that this prop is required. Is that correct? If not, can we validate the action cell renders properly in all browsers, including IE11 and Safari? Some browsers don't handle missing flex basis very well.

Reply:
Updated to the conditional below